bundles/admin/javascript-hooks/sticky-toolbar.rst

```diff
@@ -0,0 +1,24 @@
+Sticky-Toolbar
+==============
+
+The sticky-toolbar extension enables the datagrid list toolbar to sick under the
+header the page will be scrolled.
+
+To enable this extension call the following function in the `initialize` method
+of your component and disable it in the destroy method.
+
+.. code-block:: javascript
+
+    {
+        initialize: function() {
+            this.sandbox.stickyHeader.enable(this.$el);
+        },
+
+        destroy: function() {
+            this.sandbox.stickyToolbar.disable(this.$el);
+        }
```

Comment:
Wouldn't it be nice if that would be handled by Sulu somehow? Would be nice if the component just has something like this:

```javascript
{
    stickyHeader: true
}
```